### PR TITLE
chore: use same `convert_to_generic_platform_wheel.py` script as cmake

### PR DESCRIPTION
**Pull request by mayeut**
This prepares for `universal2` wheels and allows to create `py2.py3` wheels

### CODE REVIEW

**Comment by henryiii** [comment]
Why isn't there something more official for doing this sort of thing? There are other packages that have "special" output tag requirements. I've seen a regex package that needed python version _only_ (no plat/arch), we had multiple tags in cibuildwheel as a macOS hack for a little bit, etc.

File: scripts/convert_to_generic_platform_wheel.py [hunk 1]
```diff
@@ -51,6 +51,10 @@ def _convert_to_generic_platform_wheel(wheel_ctx):
     ----------
     wheel_ctx : InWheelCtx
         An open wheel context
+    py2_py3: Bool
+        Wether the pyver tag shall be py2.py3 or just the one inferred from the wheel name
```
Comment (henryiii) [inline scripts/convert_to_generic_platform_wheel.py:55]:
```suggestion
        Whether the pyver tag shall be py2.py3 or just the one inferred from the wheel name
```

**Comment by jcfr** [comment]
> Why isn't there something more official for doing this sort of thing? There are other packages that have "special" output tag requirements. I've seen a regex package that needed python version only (no plat/arch), we had multiple tags in cibuildwheel as a macOS hack for a little bit, etc.

A while back I looked into this and then work on transferring the `wheeltools` project into the scikit-build organization. 

See https://github.com/scikit-build/wheeltools/issues/2

That said, I am wondering if there is a more current effort and if we should archive or abandon `wheeltools` .... I didn't have a chance to look at the details since then.

**Comment by henryiii** [comment]
I think this is now "passing" - Travis is out of credits and AppVeyor is being weird, but CircleCI is passing.

**Comment by henryiii** [comment]
Going to give AppVeyor another chance, turned of the Travis requirement.

**Pull request merged**